README.md

```diff
@@ -500,6 +500,32 @@ server thus resetting the storage completely.
 Note that if many hundreds of megabytes of content (queue message or blob content) are stored in-memory, it can take
 noticeably longer than usual for the process to terminate since all the consumed memory needs to be released.
 
+### Use Blob Versioning
```

Comment:
As we discussed, please make the section to describe how to start Azurite with account model configure file/text. You can use blob version as a sample.
And write a file specific for blob version in https://github.com/Azure/Azurite/tree/main/docs/designs, then link the doc in Blob Version in line #1033 below.